schema/me/consignments/submission.js

```diff
@@ -0,0 +1,163 @@
+import { GraphQLObjectType, GraphQLString, GraphQLNonNull, GraphQLBoolean, GraphQLEnumType } from "graphql"
+
+export const SubmissionDimensionAggregation = new GraphQLEnumType({
+  name: "SubmissionDimensionAggregation",
+  values: {
+    CM: {
+      value: "cm",
+    },
+    IN: {
+      value: "in",
+    },
+  },
+})
+
+export const SubmissionCategoryAggregation = new GraphQLEnumType({
+  name: "SubmissionCategoryAggregation",
+  values: {
+    PAINTING: {
+      value: "Painting",
+    },
+    SCULPTURE: {
+      value: "Sculpture",
+    },
+    PHOTOGRAPHY: {
+      value: "Photography",
+    },
+    PRINT: {
+      value: "Print",
+    },
+    DRAWING_COLLAGE_OR_OTHER_WORK_ON_PAPER: {
+      value: "Drawing, Collage or other Work on Paper",
+    },
+    MIXED_MEDIA: {
+      value: "Mixed Media",
+    },
+    PERFORMANCE_ART: {
+      value: "Performance Art",
+    },
+    INSTALLATION: {
+      value: "Installation",
+    },
+    VIDEO_FILM_ANIMATION: {
+      value: "Video/Film/Animation",
+    },
+    ARCHITECTURE: {
+      value: "Architecture",
+    },
+    FASHION_DESIGN_AND_WEARABLE_ART: {
+      value: "Fashion Design and Wearable Art",
+    },
+    JEWELRY: {
+      value: "Jewelry",
+    },
+    DESIGN_DECORATIVE_ART: {
+      value: "Design/Decorative Art",
+    },
+    TEXTILE_ARTS: {
+      value: "Textile Arts",
+    },
+    OTHER: {
+      value: "Other",
+    },
+  },
+})
+
+export const SubmissionType = new GraphQLObjectType({
+  name: "Submission",
+  description: "A work to be consigned to the user",
+  fields: {
+    id: {
+      description: "Convection id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    additional_info: {
+      description: "The type of participant, e.g. Partner or User",
+      type: GraphQLString,
+    },
+    artist_id: {
+      decription: "The gravity ID for an Artist",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    authenticity_certificate: {
+      description: "Does the artwork come with an certificate of authenticity?",
+      type: GraphQLBoolean,
+    },
+    category: {
+      description: "The set in which to put the work",
+      type: SubmissionCategoryAggregation,
+    },
+    deadline_to_sell: {
```

Comment:
Up to you if you want to keep this field, but there is currently no UI for it and it's mostly in convection right now so we can accurately migrate over data that exists in rothko.